host-configs/llnl/blueos_3_ppc64le_ib_p9/clang@upstream_nvcc_c++17_no_separable.cmake

```diff
@@ -63,4 +63,4 @@ set(ENABLE_GTEST_DEATH_TESTS OFF CACHE BOOL "")
 
 # Very specific fix for working around CMake adding implicit link directories returned by the BlueOS
 # compilers to link CUDA executables 
-set(BLT_CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES_EXCLUDE "/usr/tce/packages/gcc/gcc-4.9.3/lib64/gcc/powerpc64le-unknown-linux-gnu/4.9.3;/usr/tce/packages/gcc/gcc-4.9.3/lib64" CACHE STRING "")
```

Comment:
https://github.com/LLNL/blt/blob/develop/host-configs/llnl/blueos_3_ppc64le_ib_p9/clang%40upstream_nvcc_c%2B%2B17.cmake

it is in this host-config too

Reply:
I'll fix that one.